reframe/core/environments.py

```diff
@@ -16,6 +16,9 @@ def normalize_module_list(modules):
 
     :meta private:
     '''
+    if modules is None:
+        return None
```

Comment:
I'm not sure whether returning `None` is correct here or `[]`. What I cannot understand how it came and this was called with `None`. It is called here:

```
  File "/Users/karakasv/Repositories/reframe/reframe/core/systems.py", line 282, in create
```

And there, the `site_config.get()` should return the default value, which it should be `[]`.

Reply:
Indeed querying the variable from the command line, it fails instead of returning the default.

```
./bin/reframe -C config/tresa.py --show-config=systems/0/partitions/@default/container_platforms/0/modules
```
```
./bin/reframe: no such configuration parameter found: systems/0/partitions/@default/container_platforms/0/modules
```